pom.xml

```diff
@@ -141,6 +166,19 @@
                 <artifactId>sqlite-jdbc</artifactId>
                 <version>3.28.0</version>
             </dependency>
+
+            <dependency>
```

Comment:
There is a new JTS library version. They changed the ownership of this one to https://mvnrepository.com/artifact/org.locationtech.jts/jts

Reply:
d8a218940f678db7a0693786f285cb6190657d38